### PR TITLE
Use timestamp component from psammead

### LINKED ISSUE #1648: Use @bbc/psammead-timestamp component
Following https://github.com/bbc/psammead/pull/440 we now have a `@bbc/psammead-timestamp` component

**Describe the solution you'd like**
Install and use `@bbc/psammead-timestamp`, removing the timestamp component code from simorgh

**Testing notes**
[Tester to complete]

Dev insight: Regression only

### PR DESCRIPTION
**Pull request by ghost**
Resolves #1648

**Overall change:** Use `psammead-timestamp` component instead of the one in simorgh.

**Code changes:**

- Added `@bbc/psammead-timestamp`
- Removed the `Timestamp` component from the project and used the `Timestamp` from psammead in the `TimestampContainer`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Timestamp/index.jsx
Comment (dr3) [inline src/app/containers/Timestamp/index.jsx:4]: Hey, can you pass the script into Timestamp please? Just realised it defaults to [latin](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-timestamp/src/index.jsx#L35) which we shouldn't have done 🙈 We should pass latin in explicitly, make `script` required and remove the default

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/containers/Timestamp/index.jsx
Comment (dr3) [reply src/app/containers/Timestamp/index.jsx:4]: Ive made https://github.com/bbc/psammead/issues/481 to remove the default :) doesnt need to be done before this PR, but we should anticipate it by passing in the script imo :)

Comment (ghost) [reply src/app/containers/Timestamp/index.jsx:4]: I think we should maybe wrap it in `ServiceContextConsumer` so that we can use the appropriate `script` based on the context just as we do in `Brand` for the `brandSVG`.

Comment (dr3) [reply src/app/containers/Timestamp/index.jsx:4]: We can use the `useContext` hook, but yeah sounds good :)

**Review by dr3 — APPROVED** [review]
lgtm

**Review by DenisHdz — APPROVED** [review]
LGTM

**Comment by dr3** [comment]
Popped back in 2nd code review as needs review from code owners

**Review by chris-hinds — APPROVED** [review]

**Review by jamesdonoh — APPROVED** [review]
Good stuff 👍

**Comment by david-boydell** [comment]
LGTM

**Pull request merged**
